spec/honeycomb/integrations/rails_spec.rb

```diff
@@ -109,38 +108,10 @@ def hello_error
     end
 
     describe "a standard request with an error" do
-      # TODO: remove this app setup once we return status_code and
-      # the top-level app setup matches the railtie middleware insertion
-      let(:app) do
-        Class.new(Rails::Application).tap do |app|
-          app.config.logger = Logger.new(STDERR)
-          app.config.log_level = :fatal
-          app.config.eager_load = false
-          app.config.secret_key_base = "3b7cd727ee24e8444053437c36cc66c4"
-          app.config.respond_to?(:hosts) && app.config.hosts << "example.org"
-          app.config.middleware.insert_after(
-            ActionDispatch::ShowExceptions,
-            Honeycomb::Rails::Middleware,
-            client: client,
-          )
-          app.initialize!
-
-          app.routes.draw do
-            get "/hello/:name" => "test#hello"
-            get "/hello_error" => "test#hello_error"
-          end
-        end
-      end
-
       before do
         get "/hello_error?honey=bee"
       end
 
-      # TODO: remove these lets when we add shared_examples
-      # because they're declared in
-      let(:event_data) { libhoney_client.events.map(&:data) }
-      let(:event) { event_data.first }
-
```

Comment:
the comment implies this should be replaced with the shared examples but I don't see them being added here

Reply:
It does imply that! And when I went to add [the shared examples](https://github.com/honeycombio/beeline-ruby/blob/03715e86ec9291c1a27ce0fe0e26d25bacdfde1f/spec/honeycomb/integrations/rails_spec.rb#L72-L80), I found [they were already there](https://github.com/honeycombio/beeline-ruby/blob/03715e86ec9291c1a27ce0fe0e26d25bacdfde1f/spec/honeycomb/integrations/rails_spec.rb#L139-L160).